mmcv/runner/dist_utils.py

```diff
@@ -49,7 +49,7 @@ def init_dist(launcher, backend='nccl', **kwargs):
 def _init_dist_pytorch(backend, **kwargs):
     # TODO: use local_rank instead of rank % num_gpus
     rank = int(os.environ['RANK'])
-    if IS_MLU:
+    if IS_MLU_AVAILABLE:
```

Comment:
Seems that this name is inconsistent with IS_IPU

Reply:
Their logic has some differences.